ssvsigner/ekm/local_key_manager_test.go

```diff
@@ -87,8 +87,7 @@ func TestEncryptedKeyManager(t *testing.T) {
 	require.NoError(t, err)
 
 	signerStorage := NewSignerStorage(db, networkconfig.TestNetwork.Beacon.GetNetwork(), logger)
-	err = signerStorage.SetEncryptionKey(encryptionKey)
-	require.NoError(t, err)
+	signerStorage.SetEncryptionKey(encryptionKey)
```

Comment:
We probably need to pass `EKMEncryptionKey()` instead of `EKMHash()` in this test